### DISCUSSION TITLE
Is there a way to use this API to convert the PLY file to an STL file type?

### DISCUSSION THREAD

**Discussion by EagleZRock** [dranjan/python-plyfile#74] [Q&A]
I am curious to hear the developers thoughts

**Comment by dranjan** [comment]
You can certainly use this package to parse a PLY file, but to create the other format, `plyfile` wouldn't be directly helpful. You might be better off using other tools that support both formats. Last I checked, their support for PLY wasn't as complete as `plyfile`'s, but that might not matter for your case.
